protocol/messenger_validate_requests_test.go

```diff
@@ -0,0 +1,45 @@
+package protocol
+
+import (
+	"testing"
+
+	"github.com/status-im/status-go/protocol/requests"
+
+	"github.com/stretchr/testify/suite"
+)
+
+func TestMessengerValidateRequestSuite(t *testing.T) {
+	suite.Run(t, new(MessengerValidateRequestSuite))
+}
+
+type MessengerValidateRequestSuite struct {
+	MessengerBaseTestSuite
+}
+
+func (s *MessengerValidateRequestSuite) TestSaveNewWakuNodeRequestValidate_Enrtree() {
+	alice := s.newMessenger()
```

Comment:
1. You can reuse `m` from `MessengerBaseTestSuite`
2. IF that one is not enough and you need to create another messenger instance, then use `newTestMessenger`